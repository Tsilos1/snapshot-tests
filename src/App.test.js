import { render, screen } from '@testing-library/react';
import App from './App';
// import renderer from 'react-test-renderer'


// test('renders a snapshot', () => {
//   const tree = renderer.create(<App/>).toJSON()
//   expect(tree).toMatchSnapshot()
// })

test('renders text testing this h1', () => {
  render(<App />)
  const nameElement = screen.getByText(/Testing this h1/i)
  expect(nameElement).toBeInTheDocument()
});

