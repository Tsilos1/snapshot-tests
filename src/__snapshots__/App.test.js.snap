// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="App"
>
  <div
    className="card"
    style={
      Object {
        "width": "18rem",
      }
    }
  >
    <img
      alt="Tsilos' profile image"
      className="card-img-top"
      src="profilePic.jpg"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
        Tsilos Kosbab
      </div>
      <p
        className="card-text"
      >
        Tsilos is a software development student who is learning how to do a snapshot test.
      </p>
    </div>
  </div>
</div>
`;
